VMBackup/main/backuplogger.py

```diff
@@ -32,9 +32,14 @@ def __init__(self, hutil):
         self.enforced_local_flag_value = True
         self.hutil = hutil
         self.prev_log = ''
+        self.logging_off = False
 
     def enforce_local_flag(self, enforced_local):
-        if (self.enforced_local_flag_value != False and enforced_local == False):
+        if (self.hutil.get_intvalue_from_configfile('LoggingOff', 0) == 1):
```

Comment:
can we rename as disable logging?

Reply:
no. already provided private to a customer